R/AllGenerics.R

```diff
@@ -0,0 +1,47 @@
+#' @title Save and load MS data objects to and from different file formats
+#'
+#' @aliases saveMsObject,MsBackend,ANY-method
+#' @aliases loadMsObject,MsBackend,ANY-method
+#'
+#' @description
+#'
+#' The `saveMsObject()` and `loadMsObject()` methods allow serializing and
+#' restoring/importing mass spectrometry (MS) data objects to and from language
+#' agnostic file formats. The type  and configuration of the file format is
+#' defined by the second argument to the method, `param`.
+#'
+#' - `saveMsObject(object, param)`: saves the MS data object `object` to file(s)
+#'   in a format defined by `param`.
+#'
+#' - `loadMsObject(object, param)`: `object` defines the type of MS object that
+#'   should be returned by the function and `param` the format and file name(s)
+#'   from which the data should be restored/imported.
+#'
+#' @param object for `saveMsObject()`: the MS data object to save, for
```

Comment:
this might need to be updated when we develop a method for a chromatogram object no ? should we already have a more general description ? or use 'LC-MS' object term ?